README.md

```diff
@@ -76,6 +65,10 @@ A less-recommended approach is to use the built-in [docker-ssh-env-config](https
 
 Another approach is to use the [environment agent hook](https://buildkite.com/docs/agent/hooks) to pull down the key into the container’s file system before the `git checkout` occurs. Note: the key will exist in Docker’s file system unless it is destroyed.
 
+## Customizing the image on boot
+
+The default entrypoint will look for any scripts in `/docker-entrypoint.d/` and run them. This allows for bootstrap scripts to be mounted into the agent to install things (via apk or pip) before the agent starts.
```

Comment:
What about:

```markdown
## Entrypoint customizations

The entrypoint uses `tini` to correctly pass signals to, and kill, sub-processes. Instead of redefining `ENTRYPOINT` we recommend you copy executable scripts into `/docker-entrypoint.d/`. All executable scripts in that directory will be executed in alphanumeric order.
```